changelog.d/10804.doc

```diff
@@ -0,0 +1,2 @@
+Fixed a wording mistake in the comments, which likely arose as a result of overlooking a copy-pasted segment.
+Found by @bramvdnheuvel:nltrix.net.
```

Comment:
I don't think it is useful to postulate about why it happened in the changelog (although I'm fairly certain you're correct!) We also always make changelogs a single line.

```suggestion
Fixed a wording mistake in the sample configuration. Found by @bramvdnheuvel:nltrix.net.
```